build.gradle

```diff
@@ -46,12 +43,27 @@ dependencies {
     implementation 'com.google.guava:guava'
     implementation 'com.netflix.nebula:nebula-dependency-recommender'
 
-    testCompile 'com.netflix.nebula:nebula-test'
+    testCompile platform('org.junit:junit-bom')
+    testImplementation 'com.netflix.nebula:nebula-test'
+    testImplementation "org.junit.jupiter:junit-jupiter-api"
+    testRuntimeOnly "org.junit.vintage:junit-vintage-engine"
 
     annotationProcessor "org.immutables:value"
     compileOnly "org.immutables:value::annotations"
 }
 
+tasks.withType(Test) {
```

Comment:
You likely want (or some version of)
```
testLogging {
    events "passed", "skipped", "failed"
}
```

as otherwise you get no summary when running (only what current gradle count displays)

Reply:
So I think I'm actually fine with that - you can look at the HTML report to see what's happened.

I want to keep CI output as high signal as possible, so that compiler warnings etc really stand out

Reply:
Sure - you could then just log failed? Otherwise you won’t know until ci finishes

Reply:
if tests take longer than 10 minutes, that's a bigger problem :)